lib/pmdtester/builders/diff_report_builder.rb

```diff
@@ -89,9 +89,15 @@ def build_summary_row(doc, item, base, patch, *diff)
     end
 
     def build_filename_h3(doc, filename)
-      doc.h3 do
-        doc.a(href: @project.get_webview_url(filename)) do
-          doc.text @project.get_path_inside_project(filename)
+      if filename.nil?
```

Comment:
I think, I've seen this problem locally as well now....

Reply:
ill split this fix into a separate pr